test/features/bootstrap/Quickstart/Bundle/AppBundle/Features/Context/HomepageContext.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Quickstart\Bundle\AppBundle\Features\Context;
+
+use SensioLabs\Behat\PageObjectExtension\Context\PageObjectContext;
+
+class HomepageContext extends PageObjectContext
```

Comment:
According to the docs, the dependency should be injected & not extended, did you try it? http://behat-page-object-extension.readthedocs.org/en/latest/guide/working_with_page_objects.html#injecting-page-objects-into-a-context-file